website/thunderbird/102.0/eoy/index.html

```diff
@@ -0,0 +1,109 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+ {% set active_page = "eoy" %}
+ {% extends "_includes/bedrock/base-resp.html" %}
+ {% block facebook_id %}348169218537151{# facebook.com/Thunderbird #}{% endblock %}
+ {% block twitter_id %}mozthunderbird{% endblock %}
+
+ {% block page_image %}{{ static('img/thunderbird/thunderbird-256.png') }}{% endblock %}
+ {% block page_favicon %}{{ static('img/thunderbird/favicon.ico') }}{% endblock %}
+ {% block page_favicon_large %}{{ static('img/thunderbird/favicon-196.png') }}{% endblock %}
+ {% block page_ios_icon %}{{ static('img/thunderbird/ios-icon-180.png') }}{% endblock %}
+
+ {% block page_css %}<link href="/media/css/eoy.css" rel="stylesheet" type="text/css" />{% endblock %}
+
+ {% block page_title %}{{ _('Help Keep Thunderbird Alive!') }}{% endblock %}
+
+ {% block site_header %}
+ <div id="forest-bg">
+
+</div>
+<header>
+  {% block header_content %}
+  <section id="initAppeal" class="text-left font-xl pl-8 pr-8 md:pl-16 md:pr-16 max-w-8xl mx-auto pt-12">
+    <aside>
+      <h1 class="accent-text">
+        <span class="font-light">{{ _('Help Keep')}}</span>
+        <br>
+        <span>{{ _('Thunderbird Alive!')}}</span>
+      </h1>
+      <p>
+        {{ _('Did you know that <strong>less than 1%</strong> of Thunderbird users fund all the work we do?')}}
+      </p>
+      <p>
+        {{ _('<strong>Not too long ago Thunderbird was on the verge of extinction.</strong> It’s because of donations from that small group of users that we are here today!')}}
+      </p>
+      <p>
+        {{ _('We don’t show advertisements or sell your data and we are completely funded by donations from our community.')}}
+      </p>
+    </aside>
+    <aside>
+      <span id="illustration">{{ svg('tb-donate-illustration') }}</span>
+    </aside>
+  </section>
+  {% endblock %}
+</header>
+{% endblock %}
+
+{% block email_form %}
+{% endblock email_form %}
+
+{% block content %}
+<section class="font-xl">
+  <div class="letter-container">
+    <a href="#" class="donate-banner">
```

Comment:
missing donation URL.
@ryanleesipes which donation link + utm we want here?

Reply:
This is the link we should add here: https://give.thunderbird.net/?utm_source=thunderbird-client&utm_medium=referral&utm_content=EOY22